edi_account_edifact_oca/components/listener.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2023
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo.addons.component.core import Component
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountMoveEdifactListener(Component):
+    _name = "account.move.event.listener.edifact"
+    _inherit = "base.event.listener"
+    _apply_on = ["account.move"]
+
+    def on_post_account_move(self, move):
+        edifact_backend = self.env.ref("edi_edifact_oca.edi_backend_type_edifact")
+        if (
+            not move.partner_id
```

Comment:
pls move this condition to a specific method. The direction is not needed (see the change that I proposed to the field name in the other PR).